.travis.yml

```diff
@@ -4,7 +4,7 @@ node_js:
   - "0.12"
 
 before_install:
-  - gem install scss_lint -v 0.41.0 --no-rdoc --no-ri
+  - gem install scss_lint -v 0.42.2 --no-rdoc --no-ri
```

Comment:
good point - lets leave a comment here so that next person knows why we have it and if it can be removed (I think comments are allowed in yml)